contracts/v0.2/src/SubmissionProxy.sol

```diff
@@ -407,4 +391,48 @@ contract SubmissionProxy is Ownable {
     function getAllOracles() public view returns (address[] memory) {
         return oracles;
     }
+
+    /**
+     * @notice Verify the proof
+     * @param _feed The address of the feed
+     * @param _message The hash of the message
+     * @param _proofs The proofs
+     * @return `true` if the proof is valid, `false` otherwise
+     */
+    function verifyProof(address _feed, bytes32 _message, bytes[] memory _proofs) private view returns (bool) {
```

Comment:
It's just optional and minor comment, I'd personally prefer `validateProof` for this function

Reply:
Updated at https://github.com/Bisonai/orakl/pull/1410/commits/18d8eeed7192f8c97cd24549068e26c0132c97f5